### PR TITLE
Power transmission circuits can now charge guns, but at a much slower rate.

### PR DESCRIPTION
**Pull request by ghost**
:cl:
add: Power transmission circuits can now charge guns, but at a much slower rate.
/:cl:

### CODE REVIEW

File: hippiestation/code/modules/integrated_electronics/subtypes/power.dm [hunk 1]
```diff
@@ -38,15 +38,15 @@
 /obj/item/integrated_circuit/power/transmitter/do_work()
 
 	var/atom/movable/AM = get_pin_data_as_type(IC_INPUT, 1, /atom/movable)
+	var/transfer_amount = amount_to_move
 	if(!AM)
 		return FALSE
 	if(istype(AM, /obj/item/gun/energy))
-		return FALSE
+		transfer_amount *= 0.35
```
Comment (ghost) [inline hippiestation/code/modules/integrated_electronics/subtypes/power.dm:45]: 35% is a number pulled outta you know where, btw

**Comment by WarRage333** [comment]
Bad idea. 
Its already bad as it is but people don't use it.

**Comment by k-k-karma** [comment]
blacklist advanced energy guns

**Comment by Manly-Dwarf** [comment]
the less I see circuit beepsky armies killing everyone, the better

**Comment by ghost** [comment]
Does this circuit even have any use besides cheapo inducer?

**Comment by WarRage333** [comment]
They removed the ability for inducers to power guns, and while circuits were active on tg this made some real havoc with drones that got infinite ammo.

Tho id like inducers to power guns back at a way slower rate tho, but not inside circuits.

**Comment by ghost** [comment]
would this be considered if i lowered the transfer rate to guns to 15%?

**Comment by Majick1** [comment]
Been holding off on circuit inputs, would have to say -1 though, self charging circuits would be hell

**Comment by AnthonyJNgo** [comment]
Slapping this with the close hammer, this is buffing circuits way too much.

**Pull request closed without merging**